src/pages/organize/index.tsx

```diff
@@ -0,0 +1,30 @@
+import DefaultLayout from 'utils/layout/DefaultLayout';
+import Head from 'next/head';
+import messageIds from 'features/organizations/l10n/messageIds';
+import OrganizationsDataModel from 'features/organizations/models/OrganizationsDataModel';
+import { PageWithLayout } from 'utils/types';
+import { useMessages } from 'core/i18n';
+import useModel from 'core/useModel';
+import UserOrganizations from 'features/organizations/components/OrganizationsList';
+
```

Comment:
One more thing! You also need a `getServerSideProps()` here to check that the user is authenticated.

Not having that here also seems to be causing build errors, which in turn causes the playwright test suite to fail (or not even run).

Reply:
Of course! I'll add it! thanks!